src/ip_notifier.cpp

```diff
@@ -54,11 +54,29 @@ namespace libtorrent { namespace aux
 {
 namespace
 {
+template <typename...>
+struct ip_change_notifier_none final : ip_change_notifier
```

Comment:
a trick to avoid unused warnings, is like allow to take any "Tag", but I will change that.